parseable/benchmark.sh

```diff
@@ -0,0 +1,38 @@
+# Install Dependencies
+sudo apt-get update -y
+sudo apt-get install -y rustc rustup
```

Comment:
apt-get complained for me that either `rustc` or `rustup` can be installed. I think installing just the latter will be fine.